applications/GeoMechanicsApplication/tests/dsettlement/fully_saturated_column_uniform_load/common/ProjectParameters_stage2.json

```
@@ -2,7 +2,7 @@
   "problem_data": {
     "problem_name"                    : "mesh",
     "start_time"                      : 0.0,
-    "end_time"                        : 8640000.0,
+    "end_time"                        :1.0,
```

Nitpicking: it seems there is a space missing:
```suggestion
    "end_time"                        : 1.0,
```

---

Added.